CMakeLists.txt

```diff
@@ -63,7 +63,10 @@ if (UNIX)
 	# force 32bit build for now and set necessary flags to compile code as is
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m32 -std=gnu99 -fno-omit-frame-pointer")
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m32 -std=gnu++11 -fno-omit-frame-pointer")
-	set(CMAKE_SHARED_LINKER_FLAGS "-m32")
+	set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -m32")
```

Comment:
Hmm, it seemed more intuitive and I know we use them in our files at work. But I haven't worked with cmake that much so I haven't seen that many different files yet :) If this is more common is probably better to stick with it. :+1: